eng/pipelines/libraries/build-job.yml

```diff
@@ -62,7 +62,7 @@ jobs:
         - _addtionalBuildArguments: ''
         - ${{ parameters.variables }}
         - ${{ if eq(parameters.osGroup, 'WebAssembly') }}:
-          - EMSDK_PATH: $(Build.BinariesDirectory)/emsdk
+          - EMSDK_PATH: /usr/local/emscripten
```

Comment:
This is not good for us, we need the binaries to be built against an explicit version of emscripten not a floating version which happens to be in the docker image.

Reply:
If that helps, I may remove the support for the dynamic runtime as it is not a viable solution at this point.

Reply:
It's not a floating version, it's locked to a specific version in the image and the image is pinned to a tag.

Reply:
The build issue is because emscripten wants to write to that path, will take a look.